### ISSUE TITLE
causes `<` to autocomplete instantly to <></>

### ISSUE THREAD

**Issue by asktree** [Luxcium/pop-n-lock-theme-vscode#102]
Issue Type: <b>Bug</b>

type < in a JSX element

Extension version: 3.32.7
VS Code version: Code 1.66.2 (dfd34e8260c270da74b5c2d86d61aee4b6d56977, 2022-04-11T07:49:20.994Z)
OS version: Darwin x64 20.4.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|38, 29, 20|
|Memory (System)|16.00GB (0.34GB free)|
|Process Argv|--crash-reporter-id ec34f414-0b95-4073-8ca5-da53e4b2b247|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30443607
pythontb:30283811
pythonvspyt551:30345470
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
pythonvsnew555:30457759
vscscmwlcmt:30465135
cppdebugcf:30475217

```

</details>

**Comment by ardasevinc** [comment]
YES. This drove me crazy trying to find out what was causing it.

**Comment by Luxcium** [comment]
Closed by `At/next/v3.32.9/v0.0.0` #113

**Comment by Luxcium** [comment]
Closed by `At/next/v3.32.9/v0.0.0` #113
